src/utils/KML.js

```diff
@@ -185,15 +183,37 @@ const sanitizeFeature = (feature) => {
     fill = undefined;
     stroke = undefined;
 
-    styles = [
-      new Style({
-        fill,
-        stroke,
-        image,
-        text,
-        zIndex: style.getZIndex(),
-      }),
-    ];
+    styles = (f, resolution) => {
+      /* Options to be used for picture scaling with map, should have at least
+       * a resolution attribute (this is the map resolution at the zoom level when
+       * the picture is created), can take an optional constant for further scale
+       * adjustment.
+       * e.g. { resolution: 0.123, constant: 1 / 6 }
+       */
+      if (feature.get('pictureOptions') && mapResolution) {
```

Comment:
use the feature of the parameter instead

Reply:
Done